AirCasting/SearchAndFollow/SearchLocation/SearchView.swift

```diff
@@ -0,0 +1,60 @@
+// Created by Lunar on 14/02/2022.
+//
+
+import SwiftUI
+import AirCastingStyling
+
+struct SearchView: View {
+    @StateObject var viewModel: SearchViewModel
+    @Binding var creatingSessionFlowContinues: Bool
```

Comment:
What is it doing in this view? 😅 It was added in session creating flow to dismiss all views at the end of session creation process.

Reply:
The reason for that is my previous thought that it could be useful but when I am in the process of making the second task I think it won't. I will do something with it in the next PR which will introduce the next view. Is this sound ok?